main_pipeline/tests/util_tests.py

```diff
@@ -0,0 +1,84 @@
+import unittest
+import logging
+from pathlib import Path
+
+import sys
+sys.path.append(str(Path('../')))
```

Comment:
would be better to reorganize as a python package rather than doing hacky stuff like this. I could make a PR to this PR, or you could hang tight until my Python Packaging talk next week :)

Reply:
Yeah, I thought it looked kind of sketchy when I added this. I'll wait on your presentation next week and update accordingly; I'm making this PR a draft in the meantime.

Reply:
Actually @cthoyt if this doesn't take too much work would you mind getting us started with reorganizing as a python package? Otherwise we can get started on Wednesday, but I figure might as well get a head start if we can

Reply:
What we've got so far is the `src/coronawhy_vt/` directory, where all code must go to be accessible as a package. This is already set up, ready to go, and documented.

I will make a PR for using tox and flake8 for ensuring code quality as we start adding code in here. Because I assume most code will be abandoned after its first use, so we better ensure due diligence is done on inclusion 👅

Reply:
@dnsosa ask and you shall receive... but this is going to cause a lot of friction. In my experience, people will do whatever they possibly can to avoid conforming to rules for code quality. Anyway, see: https://github.com/CoronaWhy/task-vt/pull/44